packages/devtools_app/lib/src/memory/flutter/memory_filter.dart

```diff
@@ -250,19 +217,25 @@ class SnapshotFilterState extends State<SnapshotFilterDialog>
   Widget build(BuildContext context) {
     buildFilters();
 
+    // Dialog has three main vertical sections:
+    //      - three checkboxes
+    //      - one list of libraries with at least 5 entries
+    //      - one row of buttons Ok/Cancel
+    // For very tall app keep the dialog at a reasonable height w/o too much vertical whitespace.
+    // The listbox area is the area that grows to accomodate the list of known libraries.
     return Dialog(
       shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(20.0)),
       child: LayoutBuilder(
         builder: (BuildContext context, BoxConstraints constraints) {
           return Container(
             width: MediaQuery.of(context).size.width / 3,
-            height: constraints.maxHeight < 500
+            height: constraints.maxHeight < 400
                 ? constraints.maxHeight
-                : constraints.maxHeight / 3 + 300,
+                : constraints.maxHeight / 3 + (400 * .7),
```

Comment:
this needs to be `constraints.maxHeight *.3 + (400 * .7)`, rather than
`constraints.maxHeight / 3 + (400 * .7)` to have a smooth transition.

Reply:
Ok, done.